manifests/params.pp

```diff
@@ -45,7 +45,7 @@
 
       $dhcp_config = '/etc/dhcp/dhcpd.conf'
       $dhcp_leases = '/var/lib/dhcp/dhcpd.leases'
-      $dhcp_manage_acls = false
+      $dhcp_manage_acls = true
```

Comment:
The installer will have saved this answer for users, so we'll need a migration there as well.

Reply:
Thinking about it, we probably could force acls.
That would safe us some troubles.

Reply:
But if you think we should keep it, I'll add migration :+1:

Reply:
Migration is added by theforeman/foreman-installer#576